.github/workflows/on-pr-label-update-changelog.yml

```
@@ -28,19 +36,14 @@
-        uses: peter-evans/create-pull-request@4e1beaa7521e8b457b572c090b25bd3db56bf1c5
-        with:
-          token: ${{ secrets.GH_PAT }}
-          commit-message: "chore: changelog + version bump [skip ci]"
-          branch: chore/changelog-update
-          title: "chore: update changelog and version bump"
-          body: |
-            This PR was auto-generated by CI to update the global changelog and apply version bumps via Changesets.
-          labels: automated, changelog
-          delete-branch: true
+      - name: Commit and push to PR branch
+        run: |
```

actionlint (1.7.4)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)